src/themes/BlueHintGray.ts

```diff
@@ -0,0 +1,23 @@
+import { ITheme } from "./ITheme";
+
+export const BlueHintGray : ITheme = {
+
+	DisplayName: 'BlueHintGray',
```

Comment:
You could use `Blue Hint Gray` as the display name, you don't have to concatenate. Otherwise looks good; thanks for sending the PR!